plugins/main/public/components/common/welcome/welcome.scss

```diff
@@ -44,7 +33,6 @@ span.statWithLink:hover {
   text-decoration: underline;
 }
 
-.wz-welcome-page .flex-redirect-app-links {
-  display: flex;
-  flex-grow: 1;
```

Comment:
**question**: Is unsetting the display and flex-grow styles intentional? I do not see any reference in the issue or pull request.

Reply:
The classes `.wz-welcome-page` and `.wz-module-body` are no longer in use.